README.md

```diff
@@ -85,10 +101,37 @@ atributes := map[string]interface{}{
      "likes_donuts": true,
 }, 
 user := optly.UserContext("optimizely end user", attributes)
-enabled, _ := client.IsFeatureEnabled("binary_feature", user)
+options := []decide.OptimizelyDecideOptions{decide.IncludeReasons}
+decision := userCtx.Decide("my_flag", options)
+
+var variationKey string
+if variationKey = decision.VariationKey; variationKey == "" {
+  fmt.Printf("[decide] error: %v", decision.GetReasons())
+  return
+}
+if variationKey == "control" {
+  // Execute code for control variation
+} else if variationKey == "treatment" {
+  // Execute code for treatment variation
+}
+
+}
```

Comment:
i see several syntax issues in this code when i tried it locally, can you please fix this?

Reply:
// instantiate a client
	client, err := optly.Client("SDK_KEY")
	// User attributes are optional and used for targeting and results segmentation
	attributes := map[string]interface{}{
		"state":        "California",
		"likes_donuts": true,
	}

	user := client.CreateUserContext("optimizely end user", attributes)
	options := []decide.OptimizelyDecideOptions{decide.IncludeReasons}
	decision := user.Decide("my_flag", options)
	var variationKey string
	if variationKey = decision.VariationKey; variationKey == "" {
		fmt.Printf("[decide] error: %v", decision.Reasons)
		return
	}
	if variationKey == "control" {
		// Execute code for control variation
	} else if variationKey == "treatment" {
		// Execute code for treatment variation
	}

Reply:
```
import (
  optly "github.com/optimizely/go-sdk"
)

// instantiate a client
client, err := optly.Client("SDK_KEY")
// User attributes are optional and used for targeting and results segmentation
attributes := map[string]interface{}{
  "state":        "California",
  "likes_donuts": true,
}

user := client.CreateUserContext("optimizely end user", attributes)
options := []decide.OptimizelyDecideOptions{decide.IncludeReasons}
decision := user.Decide("my_flag", options)
var variationKey string
if variationKey = decision.VariationKey; variationKey == "" {
  fmt.Printf("[decide] error: %v", decision.Reasons)
  return
}
if variationKey == "control" {
  // Execute code for control variation
} else if variationKey == "treatment" {
  // Execute code for treatment variation
}
```

Reply:
Use this instead, i have tried it locally.